[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.132.26.34:9200/
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.132.26.127:9200/
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.132.26.59:9200/
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.132.26.34:9200/_nodes
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.132.26.34:9200/_cluster/health
[32mINFO    [0m integration.ha.helpers:helpers.py:247 Continuous writes result: namespace(max_stored_id=174, count=175, last_expected_id=174)
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.132.26.34:9200/series_index/_search_shards
[32mINFO    [0m integration.ha.conftest:conftest.py:57 



The writes have been cleared.




[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Model status:

Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.5.3    unsupported  02:19:49Z

App                       Version  Status  Scale  Charm                     Channel        Rev  Exposed  Message
opensearch                         active      3  opensearch                                 0  no       
self-signed-certificates           active      1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent  Machine  Public address  Ports     Message
opensearch/0*                active    idle   1        10.132.26.34    9200/tcp  
opensearch/1                 active    idle   2        10.132.26.127   9200/tcp  
opensearch/2                 active    idle   3        10.132.26.59    9200/tcp  
self-signed-certificates/0*  active    idle   0        10.132.26.169             

Machine  State    Address        Inst id        Base          AZ  Message
0        started  10.132.26.169  juju-1ab9a0-0  ubuntu@22.04      Running
1        started  10.132.26.34   juju-1ab9a0-1  ubuntu@22.04      Running
2        started  10.132.26.127  juju-1ab9a0-2  ubuntu@22.04      Running
3        started  10.132.26.59   juju-1ab9a0-3  ubuntu@22.04      Running

[32mINFO    [0m pytest_operator.plugin:plugin.py:868 Juju error logs:

machine-1: 01:46:08 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-1: 01:46:08 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-opensearch-0: 01:46:08 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-2: 01:46:18 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-2: 01:46:18 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-opensearch-1: 01:46:18 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-0: 01:46:40 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-0: 01:46:40 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-self-signed-certificates-0: 01:46:40 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-3: 01:48:15 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-3: 01:48:15 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-opensearch-2: 01:48:15 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-opensearch-0: 01:51:05 ERROR unit.opensearch/0.juju-log certificates:3: err:  / out: keytool error: java.lang.Exception: Keystore file does not exist: /var/snap/opensearch/current/etc/opensearch/certificates/ca.p12

unit-opensearch-0: 01:51:11 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:51:12 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:51:12 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:51:12 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:51:15 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:51:18 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:51:21 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:51:24 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-1: 01:51:37 ERROR unit.opensearch/1.juju-log certificates:3: err:  / out: keytool error: java.lang.Exception: Keystore file does not exist: /var/snap/opensearch/current/etc/opensearch/certificates/ca.p12

unit-opensearch-1: 01:51:42 ERROR unit.opensearch/1.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-1: 01:51:44 ERROR unit.opensearch/1.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-2: 01:55:55 ERROR unit.opensearch/2.juju-log certificates:3: err:  / out: keytool error: java.lang.Exception: Keystore file does not exist: /var/snap/opensearch/current/etc/opensearch/certificates/ca.p12

unit-opensearch-2: 01:56:00 ERROR unit.opensearch/2.juju-log opensearch-peers:0: Cannot connect to the OpenSearch server...
unit-opensearch-2: 01:56:01 ERROR unit.opensearch/2.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-2: 01:56:02 ERROR unit.opensearch/2.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-2: 01:56:03 ERROR unit.opensearch/2.juju-log upgrade-version-a:2: Cannot connect to the OpenSearch server...
unit-opensearch-2: 01:56:04 ERROR unit.opensearch/2.juju-log node-lock-fallback:1: Cannot connect to the OpenSearch server...
unit-opensearch-2: 01:56:05 ERROR unit.opensearch/2.juju-log node-lock-fallback:1: Cannot connect to the OpenSearch server...
unit-opensearch-2: 01:56:06 ERROR unit.opensearch/2.juju-log node-lock-fallback:1: Cannot connect to the OpenSearch server...
unit-opensearch-2: 01:56:07 ERROR unit.opensearch/2.juju-log opensearch-peers:0: Cannot connect to the OpenSearch server...
unit-opensearch-2: 01:56:08 ERROR unit.opensearch/2.juju-log opensearch-peers:0: Cannot connect to the OpenSearch server...

[32mINFO    [0m pytest_operator.plugin:plugin.py:947 Forgetting model main...